drivers/ethernet/eth_e1000.c

```diff
@@ -380,7 +380,7 @@ static int ptp_clock_e1000_adjust(const struct device *dev, int increment)
 	return 0;
```

Comment:
No objection to the float -> double change. Please change the commit subject to
`net: gptp: clock sync ratio as double, not float`

Please also add information about this change to release notes in API changes section.

Reply:
Done. I hope I did it all right. Please, mark this conversation as resolved if you do, or give me some more input if I did something wrong.

Thank you very much for all your suggestions, @jukkar .